src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -101,20 +103,54 @@ function applyDateRangeFilter{{camel_case($filter->name)}}(start, end) {
 			var dateRangeInput = $('#daterangepicker-{{ str_slug($filter->name) }}').daterangepicker({
 				timePicker: false,
 		        ranges: {
-		            'Today': [moment().startOf('day'), moment().endOf('day')],
-		            'Yesterday': [moment().subtract(1, 'days'), moment().subtract(1, 'days')],
-		            'Last 7 Days': [moment().subtract(6, 'days'), moment()],
-		            'Last 30 Days': [moment().subtract(29, 'days'), moment()],
-		            'This Month': [moment().startOf('month'), moment().endOf('month')],
-		            'Last Month': [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')]
+		            '{{trans('backpack::crud.today')}}': [moment().startOf('day'), moment().endOf('day')],
+		            '{{trans('backpack::crud.yesterday')}}': [moment().subtract(1, 'days'), moment().subtract(1, 'days')],
+		            '{{trans('backpack::crud.last_7_days')}}': [moment().subtract(6, 'days'), moment()],
+		            '{{trans('backpack::crud.last_30_days')}}': [moment().subtract(29, 'days'), moment()],
+		            '{{trans('backpack::crud.this_month')}}': [moment().startOf('month'), moment().endOf('month')],
+		            '{{trans('backpack::crud.last_month')}}': [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')]
 		        },
 				@if ($filter->currentValue)
 		        startDate: moment("{{ $start_date }}"),
 		        endDate: moment("{{ $end_date }}"),
-				@endif
+                @endif
+                locale: {
+                    "format": "{{ $dateFormat }}",
+                    "applyLabel": "{{trans('backpack::crud.apply')}}",
```

Comment:
Hand-coding the translation seemed awkward to me at first. But now I see there's no translation support from the JS package itself - you have to do it yourself. So I think this is _the only way to do it_. And it was indeed something nobody complained about (yet), but that should have been done eventually. Thanks!

Reply:
Indeed. It was an easy one to solve :)

Reply:
Hmm on second thought maybe this approach was better? https://github.com/Laravel-Backpack/CRUD/pull/2111/files#r349574986

Reply:
Turns out _there is_ a translation feature for this JS library.